{
	"cve": {
		"data_type": "CVE",
		"data_format": "MITRE",
		"data_version": "4.0",
		"CVE_data_meta": {
			"ID": "CVE-2020-35132",
			"ASSIGNER": "cve@mitre.org"
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": []
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"url": "https://bugs.launchpad.net/ubuntu/+source/phpldapadmin/+bug/1906474",
					"name": "https://bugs.launchpad.net/ubuntu/+source/phpldapadmin/+bug/1906474",
					"refsource": "MISC",
					"tags": []
				},
				{
					"url": "https://github.com/leenooks/phpLDAPadmin/commit/c87571f6b7be15d5cd8b26381b6eb31ad03d28e2",
					"name": "https://github.com/leenooks/phpLDAPadmin/commit/c87571f6b7be15d5cd8b26381b6eb31ad03d28e2",
					"refsource": "MISC",
					"tags": []
				},
				{
					"url": "https://github.com/leenooks/phpLDAPadmin/compare/1.2.5...1.2.6.2",
					"name": "https://github.com/leenooks/phpLDAPadmin/compare/1.2.5...1.2.6.2",
					"refsource": "MISC",
					"tags": []
				},
				{
					"url": "https://github.com/leenooks/phpLDAPadmin/issues/130",
					"name": "https://github.com/leenooks/phpLDAPadmin/issues/130",
					"refsource": "MISC",
					"tags": []
				}
			]
		},
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "An XSS issue has been discovered in phpLDAPadmin before 1.2.6.2 that allows users to store malicious values that may be executed by other users at a later time via get_request in lib/function.php."
				}
			]
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": []
	},
	"impact": {},
	"publishedDate": "2020-12-11T05:15Z",
	"lastModifiedDate": "2020-12-11T05:15Z"
}